core/src/test/java/com/xmartlabs/base/core/observers/CompletableMainObserversWithErrorHandlingTest.java

```diff
@@ -0,0 +1,141 @@
+package com.xmartlabs.base.core.observers;
+
+import com.xmartlabs.base.core.helpers.TestingTree;
+import com.xmartlabs.base.core.rx.error.CompletableObserverWithErrorHandling;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import io.reactivex.Completable;
+import io.reactivex.plugins.RxJavaPlugins;
+import timber.log.Timber;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+public class CompletableMainObserversWithErrorHandlingTest {
+  private TestingTree testingTree = new TestingTree();
+
+  @Before
+  public void setUp() {
+    Timber.plant(testingTree);
+    RxJavaPlugins.setOnCompletableSubscribe((completable, completableObserver) ->
+        new CompletableObserverWithErrorHandling(completableObserver, ObservableResult.ERROR_HANDLING_ACTION));
+  }
+
+  @Test
+  public void callsCompletableSubscribe() {
+    Completable.fromCallable(() -> null)
+        .doOnSubscribe(disposable ->
+            Timber.tag(ObservableResult.SUBSCRIBE.toString()).i(ObservableResult.SUBSCRIBE.toString()))
```

Comment:
you can encapsule in a function which receives an `ObservableResult` in order to make the code smaller. Something like `log(ObservableResult.SUBSCRIBE)` It's only an idea.